python/beeai_framework/backend/message.py

```diff
@@ -50,8 +50,10 @@ class MessageTextContent(BaseModel):
     text: str
 
 
-class MessageImageContentImageUrl(ChatCompletionImageUrlObject):
-    pass
+class MessageImageContentImageUrl(TypedDict, total=False):
```

Comment:
Why a `TypedDict` and not a `BaseModel`? Also how does this relate the the change above?